### PR TITLE
Feature/add asset to portfolio

### PR DESCRIPTION
**Pull request by skykeyang**
Original issue mentioned "Many to Many relationship with Asset class"
Changed relationship with Asset class from "Many to Many" to "Many to One"

### CODE REVIEW

**Review by julianooii — APPROVED** [review]
lgtm ! 👍

File: src/main/java/com/backend/controller/PortfolioController.java [hunk 1]
```diff
@@ -4,34 +4,46 @@
...
 	private final IPortfolioService portfolioService;
+	private final IPortfolioAssetService portfolioAssetService;
+
 
     @Autowired
-    public PortfolioController(IPortfolioService portfolioService) {
+    public PortfolioController(IPortfolioService portfolioService, IPortfolioAssetService portfolioAssetService) {
         this.portfolioService = portfolioService;
+		this.portfolioAssetService = portfolioAssetService;
     }
```
Comment (DavidLHW) [inline src/main/java/com/backend/controller/PortfolioController.java:47]: should have consistent spacing (8 spaces?)

**Review by DavidLHW — COMMENTED** [review]
minor comment, other than that LGTM!

**Review by TimothyNgoi98 — APPROVED** [review]
thanks, LGTM!

**Pull request merged**